tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,199 @@
+import logging
+import hail as hl
+import hail.expr.aggregators as agg
+
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+from os.path import dirname
+
+from gnomad.utils.reference_genome import get_reference_genome
+
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("sex_check")
+logger.setLevel(logging.INFO) 
+def get_chr_cov(mt: hl.MatrixTable, build: str, call_rate_threshold: float=0.25, chr_name: str) -> hl.Table:
+    """
+    Calculates mean chromosome coverage.
+    
+    :param mt: MatrixTable containing samples with chrY variants
+    :param build: Reference used, either GRCh37 or GRCh38
+    :param call_rate_threshold: Minimum required call rate
+    :param chr_name: chosen chromosome, either 1, 2, 3, ..., X, or Y
+    :return: Table with coverage annotations
+    """
+
+    logger.warning("This function expects the chrY to be at index 23 and chrX to be at index 22")
+    
+    if chr_name == "Y"
+        filter_nonpar_expr = sex_mt.locus.in_y_nonpar()
+        chr_place = 23
+    elif chr_name = "X"
+        filter_nonpar_expr = sex_mt.locus.in_x_nonpar()
+        chr_place = 22
+    else:
+        try:
+            chr_place = int(chr_name) - 1
+        except:
+            logger.warning("chr_name cannot be converted to an integer")
+
+    #could utilize hail's reference genome chr names
+    chr_name= hl.get_reference(build).contigs[chr_place]
```

Comment:
```suggestion
    chr_name = hl.get_reference(build).contigs[chr_place]
```
(we try to have one space around equals sign assignments)